editor/src/main/java/net/rpgtoolkit/editor/ui/MainWindow.java

```diff
@@ -456,11 +463,11 @@ public void checkFileExtension(File file) {
       addToolkitEditorWindow(EditorFactory.getEditor(openBoard(file)));
     } else if (fileName.endsWith(".ene") || fileName.endsWith(".ene.json")) {
       addToolkitEditorWindow(EditorFactory.getEditor(openEnemy(file)));
+    } else if (fileName.endsWith(".item") || fileName.endsWith("itm.json")) {

```

Comment:
Actually already implemented what you're suggesting in develop. Editor now reads in a single default extension for each type from the properties file.

This will probably become configurable through the editor at some point.